### PR TITLE
Add documentation site content

### PR DESCRIPTION
**Pull request by jk-intuit**

### CODE REVIEW

File: docs/cli/argo-cloudops_diff.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+Diff a project target using a manifest in git
+
+```
+  argo-cloudops diff [flags]
+```
+
+### Flags
+
+```
+  -h, --help                  help for sync
```
Comment (Acepie) [inline docs/cli/argo-cloudops_diff.md:11]: Nit text

File: docs/cli/argo-cloudops_get.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+Gets status of workflow
+
+```
+  argo-cloudops get [workflow name] [flags]
+```
+
+### Flags
+
+```
+  -h, --help                  help for sync
```
Comment (Acepie) [inline docs/cli/argo-cloudops_get.md:11]: Nit text

File: docs/developers/running-locally.md [hunk 3]
```diff
@@ -107,7 +104,7 @@ env set to the same value used above.
 project and target. This returns the **ARGO_CLOUDOPS_USER_TOKEN** for the new project.
 
     ```sh
-    bash scripts/create_project.sh https://github.com/argoproj-labs/argo-cloudops.git
+    bash scripts/create_project.sh https://github.com/Acepie/argo-cloudops-example.git
```
Comment (Acepie) [inline docs/developers/running-locally.md:107]: This change should probably be reverted?

File: docs/users/coreconcepts.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
+# Core Concepts
+
+This page serves as additional information in addition to the [Argo CloudOps Architecture](/architecture).
+
+
+## Project
+
+A project is a logical collection of all deployment targets. A project contains a single property of **name**.
```
Comment (Acepie) [inline docs/users/coreconcepts.md:8]: We probably want to update this to mention manifest repo

**Review by Acepie — APPROVED** [review]
LGTM

File: docs/developers/running-locally.md [hunk 5]
```diff
@@ -1,4 +1,4 @@
-# Development Environment setup
```
Comment (grantleehoffman) [inline docs/developers/running-locally.md:1]: Does the README need an update for this file name change?

File: docs/architecture.md [hunk 6]
```diff
@@ -53,13 +53,13 @@ Additionally you can define your own frameworks in **argo-cloudops.yaml**.
 
 ## Workflow
 
-Argo CloudOps uses Argo Workflows as it's workflow engine. To execute the provided command, an Argo workflow
+Argo CloudOps uses [Argo Workflows](https://argoproj.github.io/argo-workflows/) as it's workflow engine. To execute the provided command, an Argo workflow
```
Comment (pclata) [inline docs/architecture.md:56]:
```suggestion
Argo CloudOps uses [Argo Workflows](https://argoproj.github.io/argo-workflows/) as its workflow engine. To execute the provided command, an Argo workflow
```

File: docs/faq.md [hunk 7]
```diff
@@ -0,0 +1,7 @@
+# FAQ
+
+> Argo CloudOps server returns "INTERNAL ERROR" when following logs
+
+This happens when a deployment executes for a long time. Just reissue the logs command to retry.
```
Comment (pclata) [inline docs/faq.md:5]: Should we mention that we're currently looking into the issue?

**Review by pclata — CHANGES_REQUESTED** [review]

File: docs/faq.md (hunk 7 above)
Comment (thbishop-intuit) [reply docs/faq.md:5]: maybe file a public issue and link?

File: docs/developers/running-locally.md (hunk 5 above)
Comment (jk-intuit) [reply docs/developers/running-locally.md]: renamed file and updated README.md

File: docs/faq.md (hunk 7 above)
Comment (jk-intuit) [reply docs/faq.md]: Added a public issue & linked in the faq:

https://github.com/argoproj-labs/argo-cloudops/issues/93

**Review by Acepie — APPROVED** [review]

**Review by pclata — APPROVED** [review]
LGTM

**Pull request merged**